### PR TITLE
Migrated information and music commands to slash command

### PR DESCRIPTION
**Pull request by RealYusufIsmail**
# PR Title
### Checklist

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines](/CONTRIBUTING.md).
- [x] I agree with the [license](/LICENSE).

### Changes
- [ ] Refactoring
- [x] Internal code
- [ ] External Code / Libraries
- [ ] Documentation
- [ ] Other: \_____

Closes Issue: -

## Description
I have migrated the information and music commands to slash commands meaning they will no longer use the prefix system.

### CODE REVIEW

File: src/main/java/com/github/black0nion/blackonionbot/commands/information/GuildInfoCommand.java [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
+				.setThumbnail(guild.getIconUrl())
+				.addField("name", guild.getEscapedName(), true)
+				.addField("language", guild.getLanguage() != null ? (guild.getLanguage().getName() + " (" + guild.getLanguage().getLanguageCode() + ")") : "none", true)
+				.addField("owner", guild.retrieveOwner().submit().join().getUser().getAsMention(), true)
+				.addField("serverid", guild.getId(), true)
+				.addField("rolecount", String.valueOf(guild.getRoles().size()), true)
+				.addField("membercount", String.valueOf(guild.getMemberCount()), true)
+				.addField("channelcount", String.valueOf(guild.getChannels().size()), true)
+				.addField("boostlevel", guild.getBoostTier().name(), true)
+				.addField("created", BotInformation.DATE_PATTERN.format(guild.getTimeCreated()), true));
```
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/commands/information/GuildInfoCommand.java:32]: I recently changed the way to format the date; please use `guild.getTimeCreated().format(BotInformation.DATE_TIME_FORMATTER)`

File: src/main/java/com/github/black0nion/blackonionbot/commands/information/UserInfoCommand.java [hunk 2]
```diff
@@ -74,4 +53,14 @@ private static EmbedBuilder getUserInfo(final CommandEvent cmde, final BlackUser
 		}
 		return builder;
 	}
+
+	@Override
+	public void execute(@NotNull SlashCommandEvent cmde, @NotNull SlashCommandInteractionEvent e, BlackMember member, BlackUser author, BlackGuild guild, TextChannel channel) {
+		var givenMember = e.getOption(USER, OptionMapping::getAsMember);
+
+		getUserInfo(cmde, Objects.requireNonNullElseGet(givenMember.getUser(),
+						() -> Objects.requireNonNull(e.getMember().getUser())),
```
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/commands/information/UserInfoCommand.java:62]: You can just use the passed `author` variable

File: src/main/java/com/github/black0nion/blackonionbot/commands/music/QueueCommand.java [hunk 3]
```diff
@@ -43,6 +44,6 @@ public void execute(final String[] args, final CommandEvent cmde, final MessageR
 			}
 		}
 
-		message.replyEmbeds(builder.build()).queue();
+		e.replyEmbeds(builder.build()).queue();
```
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/commands/music/QueueCommand.java:47]: Legacy code by me, you can use `cmde.reply` iirc

**Review by SIMULATAN — CHANGES_REQUESTED** [review]
Found some minor issues, but the rest looks really good, thanks a lot!

**Comment by SIMULATAN** [comment]
Looks good, I'll merge it when I'm on my PC

File: src/main/java/com/github/black0nion/blackonionbot/commands/music/VolumeCommand.java [hunk 4]
```diff
@@ -25,7 +25,7 @@ public class VolumeCommand extends SlashCommand {
 	private static final String VOLUME = "volume";
 	public VolumeCommand() {
 		super(builder(Commands.slash("volume", "Set the volume of the player")
-				.addOptions(new OptionData(OptionType.INTEGER,VOLUME, "The volume to set the player to")
+				.addOptions(new OptionData(OptionType.INTEGER,VOLUME, "The volume to set the player to", true)
```
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/commands/music/VolumeCommand.java:28]: Missing space

**Review by SIMULATAN — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/black0nion/blackonionbot/commands/music/VolumeCommand.java (hunk 4 above)
Comment (RealYusufIsmail) [reply src/main/java/com/github/black0nion/blackonionbot/commands/music/VolumeCommand.java:28]: ah i see

**Comment by RealYusufIsmail** [comment]
> Looks good, I'll merge it when I'm on my PC

I will merge as i have got your approval.

**Pull request merged**